src/schema/v2/homeView/experiments/HomeViewExperiments.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLString,
+  GraphQLFieldConfig,
+  GraphQLList,
+  GraphQLInt,
+} from "graphql"
+import {
+  FeatureFlag,
+  getExperimentVariant,
+  getFeatureFlag,
+} from "lib/featureFlags"
+import { ResolverContext } from "types/graphql"
+import { CURRENTLY_RUNNING_EXPERIMENTS } from "./experiments"
+
+const HomeViewExperiment = new GraphQLObjectType<any, ResolverContext>({
+  name: "HomeViewExperiment",
+  description: "An A/B test for home view content",
+  fields: {
+    name: {
+      type: GraphQLNonNull(GraphQLString),
+      description: "The name of the experiment",
+    },
+    enabled: {
+      type: GraphQLNonNull(GraphQLString),
+      description: "Whether the experiment is enabled",
+    },
+    service: {
+      type: GraphQLNonNull(GraphQLString),
+    },
+    description: {
+      type: GraphQLString,
+      description: "A description of the experiment",
+    },
+    variant: {
+      type: GraphQLString,
+      description: "The name of the experimental variant being shown currently",
+    },
+    variants: {
+      description: "The variants available within this experiment",
```

Comment:
This type adds a few more fields (such as `variants` here) than we had [sketched out](https://www.notion.so/artsy/Ad-Hoc-Meeting-128cab0764a0801085c6f4cc4ae6fe1c). 

I found it helpful to have access to these extra fields as I was setting this up…

![Screenshot 2024-10-23 at 11 30 15 PM](https://github.com/user-attachments/assets/6309af21-8f01-47a7-aafd-1e9cf93a223d)


…but could be persuaded to slim this down. 

Then again, if we _do_ keep this as a more generic Unleash type as suggested above, it could be nice to retain these.

Reply:
I could see a more general version of this client-facing `experiments` query field living outside of the homeView schema. Maybe this placement offers a pre-filtered selection of experiments to those only with `scope: homeview` or something like that.

That might also be useful eventually if we wanted to support dev menu awareness of all active experiments and offering override behavior, based on known variants.

Reply:
That said, maybe we could define a more generic client-facing "Experiment" type but keep the more focused placement within `homeView` to see how it feels in that context. We can always choose to expose a more generic placement later while keeping the types consistent.

The current experiment type / modeling I think is more geared towards dev/admin purposes and only used in Forque.

Reply:
- Moved `HomeViewExperiment` to the more generic `UnleashFeatureFlag` type
- Kept the `HomeViewExperiments` _field_ which continues to serve up a list of those, but only if they are `enabled: true`.

I looked into using a `scope:home-view` to magically fetch only the relevant flags. This _ought_ to be supported by Unleash's [tags](https://docs.getunleash.io/reference/tags) mechanism, but in practice I found this did not work [as advertised](https://github.com/Unleash/unleash-client-node?tab=readme-ov-file#advanced-usage)  🤷🏽, so for now we'll maintain the explicit list of experiments in `homeView/experiments.ts`